File: Proyecto Riojas/Proyecto/GrafosconListas.vb
Public Class GrafosconListas
    Class GrafosConListas : Lista 
    {
        Public override String ToString()
        {
            String grafo = "";
            Nodo1 copia = base.Head;
            While (copia!= null)
            {
                grafo += " ------> [ " + copia.Valor;

                copia = copia.Siguiente;
            }
            Return grafo;
        }
    }
End Class

File: Proyecto Riojas/Proyecto Final1/Proyecto Final1/GrafosConListas.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Proyecto_Final1
{
    class GrafosConListas :Lista 
    {
        public override string ToString()
        {
            string grafo = "";
            Nodo1 copia = base.Head;
            while (copia != null)
            {
                grafo += " ------> [ " + copia.Valor;

                copia = copia.Siguiente;
            }
            return grafo;
        }
    }
}
